### PR TITLE
[Travis] Test against 2.4

### PR DESCRIPTION
**Pull request by segiddins**

### CODE REVIEW

**Comment by segiddins** [comment]
This includes a functional change and thus requires review

File: lib/cocoapods-core/standard_error.rb [hunk 1]
```diff
@@ -98,6 +98,9 @@ def parse_line_number_from_description
       if dsl_path && description =~ /((#{Regexp.quote File.expand_path(dsl_path)}|#{Regexp.quote dsl_path.to_s}):\d+)/
         trace_line = Regexp.last_match[1]
         description = description.sub(/#{Regexp.quote trace_line}:\s*/, '')
+        if description =~ /^\s*\^\z/
+          description = description.lines[0..-3].join.chomp
+        end
```
Comment (DanToml) [inline lib/cocoapods-core/standard_error.rb]: What's the behavioural difference with 2.3 here?

Comment (DanToml) [inline lib/cocoapods-core/standard_error.rb]: (I don't have 2.4 installed locally thus can't easily check this myself rn)

**Comment by segiddins** [comment]
@DanToml the syntax error now includes the offending line and a pointer to the column

**Comment by DanToml** [comment]
LGTM with a rebase

**Comment by segiddins** [comment]
@dantoml there's no merge conflict?

**Comment by DanToml** [comment]
@segiddins ohhh. iOctocat was confused then I guess. :shipit:

**Pull request merged**